smart_open/transport.py

```diff
@@ -17,7 +17,7 @@
 
 logger = logging.getLogger(__name__)
 
-NO_SCHEME = ''
+NO_SCHEME = ""
```

Comment:
The changes in this module appear cosmetic and irrelevant, please roll them back.

Reply:
Sorry about that, I have removed the change.